From 339fab0f14b579eeabbde69c8c018123332b82c3 Mon Sep 17 00:00:00 2001
From: Jacob Chen <jacob2.chen@rock-chips.com>
Date: Thu, 22 Sep 2016 11:08:20 +0800
Subject: [PATCH] arm64: configs: add COMPAT configuration

I don't know why it was removed by former savedefconfig.
Maybe I make mistakes..

Change-Id: I4d852320c5b57ba9c72b7ef2981b6b66d76ba0b8
Signed-off-by: Jacob Chen <jacob2.chen@rock-chips.com>
---
 arch/arm64/configs/rockchip_linux_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index 5f19eb4987a1..880b0190dde6 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -36,6 +36,8 @@ CONFIG_PREEMPT_VOLUNTARY=y
 CONFIG_DEFAULT_MMAP_MIN_ADDR=32768
 CONFIG_ZSMALLOC=y
 CONFIG_SECCOMP=y
+CONFIG_ARMV8_DEPRECATED=y
+CONFIG_COMPAT=y
 CONFIG_PM_DEBUG=y
 CONFIG_PM_ADVANCED_DEBUG=y
 CONFIG_WQ_POWER_EFFICIENT_DEFAULT=y
-- 
2.35.3

